src/UserStories/UserStories.css

```diff
@@ -181,7 +202,7 @@ html {
 }
 
 .story-reply-input {
-  width: 85%;
+  width: 86%;
```

Comment:
Magic number. `flex: 1` is cleaner. (you'll just need a gap in the parent).

Reply:
Yea, much better.
I think the gaping is not needed because the distance between the wall of the story and the input in the direct icon is different, so I put some padding to match the real IG.